packages/web-sdk/src/instrumentations/performance/performanceUtils.ts

```diff
@@ -0,0 +1,129 @@
+import type { CacheType, FaroNavigationTiming, FaroResourceTiming } from './types';
+
+export function performanceObserverSupported(): boolean {
+  return 'PerformanceObserver' in window;
+}
+
+export function entryUrlIsIgnored(ignoredUrls: Array<string | RegExp> = [], entryName: string): boolean {
+  return ignoredUrls.some((url) => entryName.match(url) != null);
+}
+
+export function onDocumentReady(handleReady: () => void) {
+  if (document.readyState === 'complete') {
+    handleReady();
+  } else {
+    document.addEventListener('readystatechange', () => {
```

Comment:
Should this event listener be removed after the event?

Reply:
Good catch! let's remove after it has been executed.